scrapy/pipelines/images.py

```diff
@@ -100,14 +101,20 @@ def file_downloaded(self, response, request, info):
     def image_downloaded(self, response, request, info):
         checksum = None
         for path, image, buf in self.get_images(response, request, info):
+            content_type = Image.MIME[image.format]
+            if self.convert_type != 'PRESERVE':
```

Comment:
For imagespipeline subclasses, a test would also be needed.
The docs about custom settings is quite hard to spot. https://docs.scrapy.org/en/latest/topics/media-pipeline.html#usage-example says:

> If you have multiple image pipelines inheriting from ImagePipeline and you want to have different settings in different pipelines you can set setting keys preceded with uppercase name of your pipeline class. E.g. if your pipeline is called MyPipeline and you want to have custom IMAGES_URLS_FIELD you define setting MYPIPELINE_IMAGES_URLS_FIELD and your custom settings will be used.

So a test with `MYPIPELINE_IMAGES_CONVERT_TYPE = None`, making sure this overrides the default JPEG conversion.